### ISSUE TITLE
[Feature Request] Add delta lake file support

### ISSUE THREAD

**Issue by the-destro** [elixir-explorer/explorer#752]
Python Polars has support for delta lake files already though I can't find the function exposed in the rust package.

**Comment by josevalim** [comment]
Can you please link to the Python version of the function?

**Comment by watsy0007** [comment]
https://github.com/delta-io/delta-rs

> A native Rust library for Delta Lake, with bindings into Python

Is this the repository ?

**Comment by josevalim** [comment]
Btw, isn't delta-lake storage pretty much Parquet files? Could you access them directly instead? Writing them would be a bit more complicated though.

**Comment by watsy0007** [comment]
> Btw, isn't delta-lake storage pretty much Parquet files? Could you access them directly instead? Writing them would be a bit more complicated though.

Yes, that's correct. In our company, we integrate DuckDB, dbt, and Delta Lake with Python for business operations. I'm currently considering replacing some of these components with Elixir.
